packages/experimental/Avatar/src/useAvatar.ts

```diff
@@ -11,14 +11,17 @@ import { getHashCodeWeb } from './getHashCode';
  * @returns configured props and state for FURN Avatar
  */
 export const useAvatar = (props: JSAvatarProps): AvatarInfo => {
-  const { avatarColor, active, accessibilityLabel, activeAppearance, badge, idForColor, initials, name, src, ring, shape, ...rest } = props;
+  const { active, accessibilityLabel, accessibilityHint, activeAppearance, badge, initials, name, src, ring, shape, ...rest } = props;
 
   const showRing = active === 'active' && activeAppearance === 'ring';
   const transparentRing = !!ring?.transparent;
   const showBadge = (!active || active === 'unset') && !!badge && !!badge.status;
+  const accessibilityText = `${name || ''}${showBadge ? `, ${badge.status}` : ''}`;
 
   const imageProps: ImageProps = {
-    accessibilityLabel,
+    accessibilityLabel: accessibilityLabel || accessibilityText,
```

Comment:
Just to note: If someone adds their own a11y label, they will no longer hear the badge status. I don't think this is ideal, but one workaround would be to have the developer add the badge status to their a11yLabel if they provide one like we do on line 19. I think the right solution would be to supply the badge status to the UIA Item Status property on win32. I'll create a work item to enable that and link it here. It can be addressed in a future PR.